java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -384,6 +384,24 @@ public ColumnVector isNull() {
     return new ColumnVector(isNullNative(getNativeView()));
   }
 
+  /**
+   * Returns a Boolean vector with the same number of rows as this instance, that has
+   * TRUE for any entry that is an integer, and FALSE if its not an integer
```

Comment:
The documentation needs to clarify that this method can only be called on a string column and that "integer" here does not mean it's limited to Java's 32-bit integer type.  A true result does not mean that the value can be converted correctly (for example, converting a value too large to be represented in the type).